plugins/experimental/magick/test.sh

```diff
@@ -0,0 +1,9 @@
+#/bin/sh
+set -e -x -u;
+DIR=${DIR:=`mktemp -d;`}
+convert netlify.png $@ $DIR/netlify-2.png;
```

Comment:
We should probably use trafficserver's logo instead.

Reply:
The old or the new? I heard there is a contest going on...

Reply:
As far as I know there's no new logo yet. Current logo is in doc/static/images/press .